src/app/zap-templates/zcl/provisioning.xml

```diff
@@ -0,0 +1,163 @@
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+    <domain name="CHIP" spec="chip-0.7" dependsOn="zcl-1.0-07-5123-03" certifiable="true"/>
+    <enum name="NetworkProvisioningError" type="ENUM8">
+        <item name="Success" value="0x0"/>
+        <item name="OutOfRange" value="0x1"/>
+        <item name="BoundsExceeded" value="0x2"/>
+        <item name="NetworkIDNotFound" value="0x3"/>
+        <item name="DuplicateNetworkID" value="0x4"/>
+        <item name="NetworkNotFound" value="0x5"/>
+        <item name="RegulatoryError" value="0x6"/>
+        <item name="AuthFailure" value="0x7"/>
+        <item name="UnsupportedSecurity" value="0x8"/>
+        <item name="OtherConnectionFailure" value="0x9"/>
+        <item name="IPV6Failed" value="0xa"/>
+        <item name="IPBindFailed" value="0xb"/>
+        <item name="Label9" value="0xc"/>
+        <item name="Label10" value="0xd"/>
+        <item name="Label11" value="0xe"/>
+        <item name="Label12" value="0xf"/>
+        <item name="Label13" value="0x10"/>
+        <item name="Label14" value="0x11"/>
+        <item name="Label15" value="0x12"/>
+        <item name="UnknownError" value="0x13"/>
+    </enum>
+    <bitmap name="ShadeClosureStatus" type="BITMAP8">
+        <field name="Unencrypted" mask="0x1"/>
+        <field name="WEP-PERSONAL" mask="0x2"/>
+        <field name="WPA-PERSONAL" mask="0x4"/>
+        <field name="WPA2-PERSONAL" mask="0x8"/>
+        <field name="WPA3-PERSONAL" mask="0x10"/>
+    </bitmap>
+    <struct name="WiFiInterfaceScanResult">
+        <item name="Security" type="BITMAP8"/>
+        <item name="SSID" type="OCTET_STRING"/>
+        <item name="BSSID" type="OCTET_STRING"/>
+        <item name="Channel" type="INT8U"/>
+        <item name="FrequencyBand" type="INT32U"/>
+    </struct>
+    <struct name="ThreadInterfaceScanResult">
+        <item name="DiscoveryResponse" type="OCTET_STRING"/>
+    </struct>
+    <cluster>
+        <name>Network Provisioning</name>
+        <domain>CHIP</domain>
+        <description>TODO</description>
+        <code>0xaaaa</code>
+        <define>NWPROV_CLUSTER</define>
+        <client tick="false" init="false">true</client>
+        <server tick="false" init="false">true</server>
+        <command source="client" code="0x00" name="ScanNetworks" optional="false" cli="chip nwprov scannetworks">
+            <description>TODO</description>
+            <arg name="SSID" type="OCTET_STRING"/>
+            <arg name="Breadcrumb" type="INT64U"/>
```

Comment:
AFAICT most of the current clusters definitions use `camelCase` for the name of the arguments. Let's stay consistent.